docs/mathematics/krishnamurthy-number.mdx

```diff
@@ -0,0 +1,174 @@
+---
+title: Krishnamurthy Number
+slug: /mathematics/krishnamurthy-number
+---
+
+## Introduction
+
+Krishnamurthy Number is a number whose sum of the factorial of digits is equal to the number itself. The Number is named after the Indian speaker U. G. Krishnamurti. The number is also known as Strong number or Peterson number.
+
+## Explanation
+
+Consider a number `n` = `pqr` (where p, q, and r are its individual digits). Then it is said to be a Krishnamurthy number if and only if,
+
+```
+p! + q! + r! = pqr = n
+```
+
+For Example, consider n = 145
+
+```
+1! + 4! + 5! = 145
+```
+
+Hence 145 is a Krishnamurthy number.
+
+## Algorithm
+
+Let us assume that the given number be `num`,
+
+-   Store the value of the given number in a temporary variable `temp`.
+-   Declare a variable `sum` and set its value to 0.
+-   Begin a While loop till `temp > 0`
+    -   Set `rem = temp % 10`
+    -   Set `sum = sum + factorial(rem)`
+    -   Divide temp by 10.
+-   Check if the value of `temp` is equal to `num`, If YES return true, Else return false
```

Comment:
Keep `YES` and `false` in bacticks. Also keep them consistent. If you are using YES, then use NO for else wise. Otherwise the true/false pair